src/Commands/Setup.php

```diff
@@ -98,6 +100,10 @@ protected function copyAndReplace(string $file, array $replaces): void
     {
         $path = "{$this->sourcePath}/{$file}";
 
+        if ($file === 'Config/Security.php') {
+            $path = "{$this->distPath}/{$file}";
+        }
```

Comment:
This `if` is not good. 

The `copyAndReplace()` method does:
1. read a source file
2. replace it
3. write it to dest path

But this change will break the behavior.
It is unpredictable for devs if they read the code carefully.

Conclusion: you should not use this method for `Config/Security.php`.

Reply:
@kenjis for the step by step explanation, thank you, I like this type of explanation.

I changed the code as follows:

```php
    private function setSecurityItem(): void
    {
        $file     = 'Config/Security.php';
        $replaces = [
            'public $csrfProtection = \'cookie\';' => 'public $csrfProtection = \'session\';',
        ];

        $path      = $this->distPath . $file;
        $cleanPath = clean_path($path);

        if (!is_file($path)){
            CLI::write(CLI::color('  Not found: ', 'red') . $cleanPath);

            return;
        }
        
        $content = file_get_contents($path);
        $output = $this->replacer->replace($content, $replaces);

        // check $csrfProtection = 'session' 
        if ($output === $content) {
            CLI::write(CLI::color('  Security Setup: ', 'green') . 'Everything is fine.');

            return;
        }

        if (write_file($path, $output)) {
            CLI::write(CLI::color('  Update: ', 'green') . "We have updated file '{$cleanPath}' for security reasons.");
        } else {
            CLI::error("  Error updating file {$cleanPath}.");
        }
    }
```

Is it understandable and readable enough?

Reply:
Okay, it is the good first step. Now we have only one Config file that will be updated directly, it is fine.

Can you change the method name to `setSecurityCSRF`?

Isn't it error?
```php
CLI::write(CLI::color('  Not found: ', 'red') . $cleanPath);
```
If there is no `Config/Security.php`, it cannot set `session` for CSRF protection,
and Shield will not work.

Reply:
> Can you change the method name to `setSecurityCSRF`?

**Done.**

> Isn't it error?

Yes! In fact, checking the existence of file `Config/Security.php` is a bit exaggerated. However, I considered the worst possible situation.
**Done.**